src/doc/en/installation/conda.rst

```diff
@@ -156,5 +131,5 @@ To build the documentation, use::
 
   You can update the conda lock files by running
   ``.github/workflows/conda-lock-update.py`` or by running
-  ``conda-lock --platform linux-64 --filename environment-3.11-linux.yml --lockfile environment-3.11-linux.lock``
+  ``conda-lock --platform linux-64 --filename environment-3.12-linux.yml --lockfile environment-3.12-linux.lock``
```

Comment:
I updated some strings in this file. Not that it really matters but maybe it is going to keep the commands valid literally for a little bit longer.